hw03_karimh.py

```diff
@@ -8,20 +8,111 @@
 
 import turtle
 
-def function_1():
-    "include docstring"
+#Assigning name to the turtle
+Heer = turtle.Turtle()
```

Comment:
There should not be code outside of `main()`. This likely needs to be inside main, and passed into functions as parameters.

Reply:
`Heer` is still outside of the functions.